SessionMessagingKit/Utilities/ObservableKey+SessionMessagingKit.swift

```diff
@@ -85,6 +85,12 @@ public extension ObservableKey {
     static let messageRequestDeleted: ObservableKey = "messageRequestDeleted"
     static let messageRequestMessageRead: ObservableKey = "messageRequestMessageRead"
     static let messageRequestUnreadMessageReceived: ObservableKey = "messageRequestUnreadMessageReceived"
+    
+    // MARK: - Disappearing Messages
+    
+    static func disappearingMessagesConfigUpdated(_ id: String) -> ObservableKey {
+        ObservableKey("disappearingMessagesConfigUpdated-\(id)", .disappearingMessagesConfigUpdate)
```

Comment:
The pro integration branch has added an event for this (it's a `ConversationEvent.Change` rather than a separate event) so can probably remove this once that is merged and swap over